.gitattributes

```diff
@@ -0,0 +1,3 @@
+*.js linguist-detectable=false
```

Comment:
Don't need a gitattributes - since we donot want to force any language specific statistic

Reply:
Sorry, didn't mean to add to master branch